src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+import users from '../../api/users';
+
+type Props = {
+  addTodo: (title: string, userId: number) => void;
+};
+
+export const AddTodoForm = (props: Props) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserId, setHasUserId] = useState(false);
+
+  const handleSudmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
There is a typo in the function name `handleSudmit`. It should be `handleSubmit` to accurately reflect its purpose.